### ISSUE TITLE
docker-compose -f=docker-compose.yml fails with IOError

### ISSUE THREAD

**Issue by josephearl** [docker/compose#3305]
The usage instructions for Docker Compose (docker-compose version 1.7.0rc1, build 1ad8866) says:

```
Define and run multi-container applications with Docker.

Usage:
  docker-compose [-f=<arg>...] [options] [COMMAND] [ARGS...]
  docker-compose -h|--help
```

Running `docker-compose -f=docker-compose.yml` fails with the error:

```
ERROR: .IOError: [Errno 2] No such file or directory: u'./=docker-compose.yml'
```

Using `--file` instead of `-f` works (i.e. docker-compose --file=docker-compose.yml).

It looks like in the case `-f=docker-compose.yml` that Docker Compose is incorrectly treating the `=` as part of the argument value.

**Comment by johnharris85** [comment]
This is actually the way that Docopt (the underlying cli argument parsing library) works. It does not allow '=' in the short form of option (-o).

> <h3>-o --option</h3>
> 
> <p>Words starting with one or two dashes (with exception of "<code>-</code>", "<code>--</code>"
> by themselves) are interpreted as short (one-letter) or long options,
> respectively.</p>
> 
> <ul>
> <li>Short options can be "stacked" meaning that <code>-abc</code> is equivalent to
> <code>-a -b -c</code>.</li>
> <li>Long options can have arguments specified after space or equal "<code>=</code>" sign:<br>
> <code>--input=ARG</code> is equivalent to <code>--input ARG</code>.</li>
> <li>Short options can have arguments specified after <em>optional</em> space:<br>
> <code>-f FILE</code> is equivalent to <code>-fFILE</code>.</li>
> </ul>

Seems like doc string just needs to be changed. Fixed by #3335

**Comment by basickarl** [comment]
I'm receiving this issue: https://github.com/docker/compose/issues/5757
